lib/customer.rb

```diff
@@ -0,0 +1,32 @@
+class Customer
+
+  attr_reader :name, :tel_no
+
+  def initialize name, tel_no
+    @name = name
+    @tel_no = tel_no
+  end
+
+  def check_bill restaurant
+    find_order restaurant
+    "Expected bill of $#{calculate_bill}, was charged $#{@restaurant_bill}"
+  end
+
+  private
+
+  def find_order restaurant
+    @my_order = nil
+    restaurant.orders.clone.select.with_index do |ord, ind|
+      if ord.customer === self
```

Comment:
Style/CaseEquality: Avoid the use of the case equality operator `===`. (https://github.com/bbatsov/ruby-style-guide#no-case-equality)